package.json

```diff
@@ -29,33 +29,33 @@
     "screenfull": "3.0.0"
   },
   "devDependencies": {
-    "browserify": "13.0.0",
-    "bulk-require": "^1.0.0",
+    "browserify": "14.5.0",
+    "bulk-require": "^1.0.1",
```

Comment:
I dont remember what the different symbols like '^' mean but feel free to change it to just the version number if it will be safer. So we don't run the risk of a build not working because it installs a version that is slightly different.